### ISSUE TITLE
Add logo to welcome page / albums headers

### ISSUE THREAD

**Issue by pricelessrabbit** [LycheeOrg/Lychee#544]
It will be cool to have a way to insert a custom logo image / svg in the welcome page and header of albums. If you approve the idea i can try to create a pull request

**Comment by ildyria** [comment]
I was considering adding a header to albums at some point (including the description of the album).

I am totally up fine with this. However i have two requests:

1. Please do 2 PR for that. I mean that the modification of the landing page must be in a separate PR than the addition of the header to the album.

2. Make the header for albums configurable: either let the user chose the picture or define some rules, such as the starred picture or the latest one.... It also needs to be possible to disable it.

If you have any questions. Feel free to ask them here or on our gitter: https://gitter.im/LycheeOrg/Lobby :)

**Comment by ariznaf** [comment]
Great, I was thinking of the same enhancement.

But it would be greater to be able to select an album (or smart album) to use as the logo banner in the upper space of the landing page, and the images in that album would be changed and animated like a slide player,  with a zoom effect  (similar to chevereto were you can use several images and it animates transitions).

In an album it would be great to have a banner logo too, but it would be better to be able to use the featured image in it (or its sub-albums) instead of just one image.

**Comment by ildyria** [comment]
@ariznaf

I have to admit, Chevereto looks good. :x

To be honest, unless works on it, this feature will probably have to wait for the rewrite of the front-end... which will not happen before next year. I am currently writing my PhD thesis and my free time is gone. :(

**Comment by ariznaf** [comment]
Thanks for your quick response.
We will wait, important things come first, of course.

Yes Chevereto looks great, but lacks several things that Lychee has (better metadata managment, put your photos in a map, sub albums, smart albums...).

I camo from koken, and it had several good ideas too (like pages and custom menus).

I have read here in some enhancement request that there were pages and menus (there is a page controller) but I cannot find where you can activate it.

I am using the docker install in a QNAP NAS (lycheeof/Lychee) I am not sure if I am missing something in the install (like the lychee-front as public/Lychee-front directory is empty in the docker container).

**Comment by ildyria** [comment]
First of all:

> (like the lychee-front as public/Lychee-front directory is empty in the docker container).

No Lychee-front contains the source code of the front-end, this is useful to develop it but not to use it. We have a script which concatenate all the files and minimize it. Nothing wrong here.

> I have read here in some enhancement request that there were pages and menus (there is a page controller) but I cannot find where you can activate it.

Indeed there is a preliminary implementation of pages... but no back-end to use it. You will have to edit everything in your SQL database directly (e.g. with phpmyadmin), there is also already a markdown parser integrated in the php side to help give form to your text.

So far the functionalities are just to add a menu bar on the landing pages and additional pages in parallel to the landing page. You can check my own photography website to get an idea of the result e.g.: https://photography.viguier.nl/about (this page is definitively not up to date. :rofl: )

**Comment by ariznaf** [comment]
Thank you, for the update.

So, I am not missing something (something that is already implemented in the standard latest distribution).

That is exactly what I would like to get: a beautiful and configurable landing page, with an upper menu, and a few pages to describe myshelf and other stuff.

But I will have to wait for upcoming versions of Lychee that hopefully will implement that in the backend, as I won't know how to do that modifications and would break something.

Besides, I am running Lychee in a docker container with sqlite in order to isolate it from the system and simplify things (like backups, which I will have just to backup a file to get the database backed up).
